### DISCUSSION TITLE
Brak aktualizacji listy Polskie Filtry Wewnętrzne

### DISCUSSION THREAD

**Discussion by Brissinger** [MajkiIT/polish-ads-filter#17373] [Q&A] [answered]
Cześć,
chciałbym się dowiedzieć co się stało z listą "Polskie Filtry Wewnętrzne"? Czy zostały zmergowane z inną listą?

**Comment by hawkeye116477** [comment, marked as answer]
@KonoromiHimaries zrezygnował z jej rozwijania.

**Comment by ghost** [comment]
czeka na wsparcie https://github.com/yourduskquibbles/webannoyances/issues/238

**Reply by krystian3w** [reply]
to tylko nawigacje, nie wytnie reklam jakich MajkiIT nie tyka.

**Reply by seba2282** [reply]
> czeka na wsparcie [yourduskquibbles/webannoyances#238](https://github.com/yourduskquibbles/webannoyances/issues/238)

@KonoromiHimaries, czyli jak bardzo lista zostanie okrojona przed dodaniem wsparcia do Web Annoyances Ultralist?

**Reply by krystian3w** [reply]
Jak na podstawie podlinkowanego pliku, to tylko do latających nawigacji (tych co się przylepiają do górnej krawędzi ekranu), czyli `WAU` raczej nie wytnie reklamy Office na `centrumxp.pl`.

Jak się nie boisz, możesz próbować ją po angielsku zgłosić jako `self-promotion` i napisać że polska lista ani EasyList Polish nie jest chętna do ukrycia jej.

**Reply by seba2282** [reply]
> Jak na podstawie podlinkowanego pliku, to tylko do latających nawigacji (tych co się przylepiają do górnej krawędzi ekranu), czyli `WAU` raczej nie wytnie reklamy Office na `centrumxp.pl`.
> 
> Jak się nie boisz, możesz próbować ją po angielsku zgłosić jako `self-promotion` i napisać że polska lista ani EasyList Polish nie jest chętna do ukrycia jej.

Jak jeszcze były filtry wewnętrzne, to dłuższy czas one także tego nie blokowały znaczy reklam na centrumxp. Więc nie rozumiem, czemu cały czas przywołujesz ten link. 😂

**Reply by krystian3w** [reply]
Bo to raczej największy portal, reszta to były jakieś małe internetowe sklepiki przykładowo.

**Reply by seba2282** [reply]
@krystian3w możesz sprawdzić na centrumxp.pl czy u ciebie też działa usunięcie reklamy z office?
www.centrumxp.pl##section.sidebar-news-grid:nth-of-type(1)

**Reply by krystian3w** [reply]
Działa możesz robić issue na https://github.com/yourduskquibbles/webannoyances/issues/new i użyj słowa `self-promotion`.

**Reply by seba2282** [reply]
@krystian3w I to na baner na głównej centrumxp.pl : 

||www.centrumxp.pl/api/Images/Bannery/banner_OnexStore_378813.jpg$image 

Chodzi, co on wyświetla na samej górze. Ale nie wiem czy to blokować także.

**Reply by seba2282** [reply]
![Bez tytułu](https://user-images.githubusercontent.com/16944478/99146185-14afe400-2676-11eb-9678-7c300e97030e.png)
www.centrumxp.pl##.slick-slider.slick-initialized.main-banner--images.main-banner to wycina górny baner, ale ucina też trochę z artykułów poniżej. @krystian3w

**Reply by krystian3w** [reply]
trzeba by zobaczyć `:remove()` (FX i Chrome) na slajd lub html filering (Firefox **only**).

---

```css
centrumxp.pl##a[href*="OnexStore" i] > img[src*="OnexStore" i]:upward(p.slick-slide):remove()
centrumxp.pl##.slick-dots li:nth-of-type(4):remove()
```

**Reply by seba2282** [reply]
@krystian3w Te banery z góry można by całkiem usunąć. Ta twoja reguła usuwa tylko onexstore. Jeżeli wiesz co mówię. Bez tych banerów centrumxp lepiej wygląda.

**Reply by seba2282** [reply]
@krystian3w Po twojej regule (teraz zauważyłem) jak masz te banery na górze, zacząłem je przewijać za pomocą strzałek i gdzie był onexstore, to jest puste pole (jako 4 baner). Jak to ukryć?

**Reply by krystian3w** [reply]
Może skopiowałeś stary bez `:remove()`, jak nie to slider jest inteligenty i wykrywa że brakuje mu jednej klatki - wtedy trzeba sprawdzić w Firefox HTML filtering a w Edge będzie wyciekać.

```css
! Firefox
centrumxp.pl##^a[href*="OnexStore" i] > img[src*="OnexStore" i]:upward(p.slick-slide)
centrumxp.pl##^.slick-dots li:nth-of-type(4)


! AdGuard na Windows / Android / macOS i dodatek ich Firefokosowy
centrumxp.pl##^p.slick-slide:nth-of-type(1)[tag-content="OnexStore"]
centrumxp.pl##^.slick-dots li:nth-of-type(4)
```

**Reply by seba2282** [reply]
@krystian3w na 100% skopiowałem dobrze i testowałem na edge chromium. Może Chromium coś nie domaga? A jak wygląda sytuacja na chrome?

**Reply by krystian3w** [reply]
Nie wiem, w Firefox niby `:remove()` działało dobrze.

**Comment by Brissinger** [comment]
Dzięki za info :)

**Reply by krystian3w** [reply]
Ale tam niby tylko zostały zlecone nawigacje ruchome, reklamy jakich MajkiIT nie wycina **wyciekną**.

**Comment by seba2282** [comment]
@KonoromiHimaries @krystian3w co już wiadomo? Znaczy jak teraz będzie blokowane to, czym zajmowały się filtry wewnętrzne?

**Reply by hawkeye116477** [reply]
Będą tylko przypinane nagłówki/menu. A wewnętrzne w zestawie chyba miały jeszcze chowanie donacji, ofert promocyjnych, np na x-komie ten slider `świętuj z nami, urodzinowa wyprzedaż`, były też zaincludowane jakieś listy do prywatności, schowane linki do własnych aplikacji, sklepów, partnerów, statyczne komunikaty ciasteczkowe, puste sekcje, elementy sponsorowane, katalogi firm, badania. Ogólnie rzecz mówiąc, ja bym powiedział, że ta lista była do wszystkiego...

**Comment by seba2282** [comment]
@KonoromiHimaries szkoda, że zakończyłeś żywot polskich filtrów wewnętrznych, bo na prawdę były przydatne. Natomiast autor od Ultralist ma sporo reguł, które się trochę gryzą ze wszystkim. On dodał też reguły dla PL? Bo mówiłeś, że on cię wspiera, albo ja źle widziałem.

**Reply by xxcriticxx** [reply]
@seba2282 zawsze mozesz zrobic wlasnego forka Matt repo i kontynuowac jego prace
